components/List/src/ListItemText.tsx

```diff
@@ -29,15 +17,16 @@ export interface ListItemTextProps extends BaseProps {
 /**
  * Primary UI component for user interaction
  */
-export const ListItemText: React.FC<ListItemTextProps> = (props: ListItemTextProps) => {
-  const primary = <Typography classes={listitemtextprimary_classes}>{props.primary}</Typography>;
-  const secondary = props.secondary ? (
-    <Typography classes={listitemtextsecondary_classes}>{props.secondary}</Typography>
-  ) : undefined;
+export const ListItemText: React.FC<ListItemTextProps> = ({ primary, secondary, ...props }: ListItemTextProps) => {
+  const rootClassNames = clsx('denhaag-list__item-text', {
+    'denhaag-list__item-text--with-secondary': secondary,
+  });
+
   return (
-    <MaterialListItemText {...props} primary={primary} secondary={secondary} disableTypography={true} classes={classes}>
-      {props.children}
-    </MaterialListItemText>
+    <div {...props} className={rootClassNames}>
+      {primary && <span className="denhaag-list__item-text-primary">{primary}</span>}
+      {secondary && <span className="denhaag-list__item-text-secondary">{secondary}</span>}
```

Comment:
The secondary item text is not styled according to figma. (Figma has all uppercase and grey-3)